proto/sequencerapis/astria/sequencer/v1alpha1/transaction.proto

```diff
@@ -127,3 +129,32 @@ message FeeAssetChangeAction {
     bytes removal = 2;
   }
 }
+
+// `InitBridgeAccountAction` represents a transaction that initializes
+// a bridge account for the given rollup on the chain.
+//
+// The sender of the transaction is used as the owner of the bridge account
+// and is the only actor authorized to transfer out of this account via
+// a `TransferAction`.
+message InitBridgeAccountAction {
+  bytes rollup_id = 1;
```

Comment:
Can you add comments on these fields? Specifically constraints if there are any?